packages/cli/src/controllers/cta.controller.ts

```diff
@@ -0,0 +1,23 @@
+import express from 'express';
+import { Authorized, Get, RestController } from '@/decorators';
+import { AuthenticatedRequest } from '@/requests';
+import { CtaService } from '@/services/cta.service';
+
+/**
+ * Controller for Call to Action (CTA) endpoints. CTAs are certain
+ * notifications/messages that are shown to users in the UI.
+ */
+@Authorized()
+@RestController('/cta')
+export class CtaController {
```

Comment:
Do we expect these messsages to always have a CTA? If not, I'd generalize as `NotificationsController`.

Reply:
Currently this is the only notification/CTA/message there is. I didn't want to start generalizing this too much since we don't have any known other use cases for this as of now.